src/components/search.js

```diff
@@ -68,47 +71,40 @@ export const Search = ({ hideSudreddit, showSubreddit }) => {
 
   const handleMouseEnter = () => {
     setShowSubreddits(true);
+    size.width < 400 && onFocusChange(false);
```

Comment:
should it not be true ? when mouse enters, we are focusing, so onFocusChange should send new value as true